utils/build/docker/dotnet_http/parametric/Program.cs

```diff
@@ -0,0 +1,27 @@
+using System.Diagnostics;
+using ApmTestApi.Endpoints;
+
+// Force the initialization of the tracer
+_ = Datadog.Trace.Tracer.Instance;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddEndpointsApiExplorer();
```

Comment:
It doesn't look like the Api Explorer isn't used in the app, can we remove it? I looked at this blog post conveniently written by Andrew and it doesn't seem like you're injecting the information anywhere: https://andrewlock.net/introduction-to-the-apiexplorer-in-asp-net-core/